Packs/SecLyticsAugurIOCsFeed/Integrations/SeclyticsFeed/SeclyticsFeed.py

```diff
@@ -0,0 +1,230 @@
+        return indicator_list
+
+    @staticmethod
+    def find_indicator_type(indicator: str) -> str:
+        """
+        Get the type of the indicator.
+
+        Args:
+            indicator (str): The indicator whose type we want to check.
+
+        Returns:
+            str: The type of the indicator.
+        """
+        if re.match(urlRegex, indicator):
+            return FeedIndicatorType.URL
+        elif ip_type := FeedIndicatorType.ip_to_indicator_type(indicator):
+            return ip_type
+        elif re.match(sha256Regex, indicator):
+            return FeedIndicatorType.File
+        else:
+            return FeedIndicatorType.Domain
+
+    def create_indicators_from_response(self, response: list, feed_tags: list, tlp_color: Optional[str]) -> list:
+        """
+        Creates a list of indicators from a given response
+        Args:
+            response: List of dict that represent the response from the api
+            feed_tags: The indicator tags
+            tlp_color: Traffic Light Protocol color
+
+        Returns:
+            List of indicators with the correct indicator type.
+        """
+        parsed_indicators = []  # type:List
+
+        for indicator in response:
+            if indicator:
+                indicator_type = self.find_indicator_type(indicator)
+
+                # catch ip of the form X.X.X.X:portNum and extract the IP without the port.
+                if indicator_type in [FeedIndicatorType.IP, FeedIndicatorType.CIDR,
+                                      FeedIndicatorType.IPv6CIDR, FeedIndicatorType.IPv6] and ":" in indicator:
+                    indicator = indicator.split(":", 1)[0]
+
+                indicator_obj = {
+                    "type": indicator_type,
+                    "value": indicator,
+                    "rawJSON": {
+                        'value': indicator,
+                        'type': indicator_type,
+                        'service': 'Daily Threat Feed'
+                    },
+                    'fields': {'service': 'Daily Threat Feed', 'tags': feed_tags}
+                }
+                if tlp_color:
+                    indicator_obj['fields']['trafficlightprotocol'] = tlp_color
+
+                parsed_indicators.append(indicator_obj)
+
+        return parsed_indicators
+
+    def build_iterator(self, feed_tags: List, tlp_color: Optional[str], limit=None, offset=None):
+        """Builds a list of indicators.
+        Returns:
+            list. A list of JSON objects representing indicators fetched from a feed.
+        """
+        response = self.daily_http_request()
+        parsed_indicators = self.create_indicators_from_response(response, feed_tags,
+                                                                 tlp_color)  # list of dict of indicators
+
+        # for get_indicator_command only
+        if limit:
+            parsed_indicators = parsed_indicators[int(offset): int(offset) + int(limit)]
```

Comment:
in case the limit is defined, it's better to cut the result at the start of the process, in the function `create_indicators_from_response` or even in `daily_http_request` instead to get, create indicators and just throw them in the last point